youtube_dl/extractor/dvtv.py

```diff
@@ -145,32 +161,31 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, video_id)
 
-        # live content
-        live_item = self._search_regex(
-            r'(?s)embedData[0-9a-f]{32}\.asset\.liveStarter\s*=\s*(\{.+?\});',
-            webpage, 'video', default=None)
-
-        # single video
-        item = self._search_regex(
-            r'(?s)embedData[0-9a-f]{32}\[["\']asset["\']\]\s*=\s*(\{.+?\});',
-            webpage, 'video', default=None)
-
-        if item:
-            return self._parse_video_metadata(item, video_id, live_item)
-
         # playlist
         items = re.findall(
-            r"(?s)BBX\.context\.assets\['[0-9a-f]{32}'\]\.push\(({.+?})\);",
+            r"(?s)playlist\.push\(({.+?})\);",
             webpage)
-        if not items:
-            items = re.findall(r'(?s)var\s+asset\s*=\s*({.+?});\n', webpage)
 
         if items:
+            timestamp = parse_iso8601(self._html_search_meta(
+                'article:published_time', webpage, 'published time', default=None))
+
             return {
                 '_type': 'playlist',
                 'id': video_id,
                 'title': self._og_search_title(webpage),
-                'entries': [self._parse_video_metadata(i, video_id) for i in items]
+                'entries': [self._parse_video_metadata(i, video_id, timestamp) for i in items]
             }
 
+        # single video
+        item = self._search_regex(
+            r'(?s)BBXPlayer.setup\((.+?)\);',
+            webpage, 'video', default=None)
+
+        # remove function calls (ex. htmldeentitize)
+        item_without_funcs = re.sub(r'\w+?\((.+)\)', r'\1', item)
```

Comment:
should be handled someway in `js_to_json`, at least add a `TODO` to indicate that this should be fixed in a general way.